src/ks-app/src/app/containers/drag-and-drop/dnd.component.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component } from '@angular/core';
+
+import { Inventory } from '../data/inventory';
```

Comment:
We've been importing every public member of the API for the ks-app, to ensure we catch possible issues with something not being exported properly. Please add the other objects as inert properties of the component (see others for examples)